graylog2-web-interface/src/views/components/messagelist/MessageFields/MessageFieldsEditModal.tsx

```diff
@@ -70,8 +75,16 @@ const MessageFieldsEditModal = ({ toggleEditMode }) => {
     toggleEditMode();
   }, [resetFavoriteFields, toggleEditMode]);
 
+  const _toggleEditMode = useCallback(() => {
+    sendTelemetry(TELEMETRY_EVENT_TYPE.SEARCH_WIDGET_ACTION.CHANGE_MESSAGE_FAVORITE_FIELDS_EDIT_CANCELED, {
+      app_action_value: 'canceled',
+    });
+
+    return toggleEditMode();
+  }, [sendTelemetry, toggleEditMode]);
+
   return (
-    <Modal onHide={toggleEditMode} show rootProps={{ zIndex: 1030 }}>
+    <Modal onHide={_toggleEditMode} show rootProps={{ zIndex: 1030 }}>
```

Comment:
1.) We should not have to deal with a custom `z-index` here.
2.) If there is not enough vertical screen estate, two scrollbars will be shown:

<img width="1197" height="917" alt="Image" src="https://github.com/user-attachments/assets/2a70c6c7-37d6-462e-98f5-c2f013e1ec6a" />

Can we make sure that the buttons will always be shown and the field list - which is scrollable - is shrinking?

Reply:
@dennisoelkers https://github.com/Graylog2/graylog2-server/pull/24203 it can be something like that